test/sql-luatest/ghs_119_too_long_mem_values_test.lua

```diff
@@ -1,6 +1,10 @@
-local server = require('luatest.server')
+-- tags: parallel
+
+local server = require('luatest.server') -- luacheck: ignore
```

Comment:
`luacheck: ignore` seems to be redundant.

Reply:
Fixed, thanks:

```diff
--- a/test/sql-luatest/ghs_119_too_long_mem_values_test.lua
+++ b/test/sql-luatest/ghs_119_too_long_mem_values_test.lua
@@ -1,6 +1,6 @@
 -- tags: parallel
 
-local server = require('luatest.server') -- luacheck: ignore
+local server = require('luatest.server')
 local t = require('luatest')
 
 local is_asan = require('tarantool').build.asan